### DISCUSSION TITLE
Control or Get Information in Node RED about Physical Devices

### DISCUSSION THREAD

**Discussion by Ganc1to** [andrei-tatar/node-red-contrib-smartnora#143] [Q&A] [answered]
I mean, get in node Red info about other devices linked to the house, for example STF Home ligth, know the state of that device in nodered or control it.

**Comment by andrei-tatar** [comment, marked as answer]
It's not possible to get information from other services linked to Google Home. The scope of Smart NORA is not to control other integrations but to provide a way to make your own integration
